inst/@sym/taylor.m

```diff
@@ -117,28 +120,40 @@
     end
   end
 
+  if isa(n, 'sym')
+    n = double(n);
+  end
+
```

Comment:
n can't have symbols (x, y, z.....), maybe use findsymbols? or remove it and lets python handle an error?